### ISSUE TITLE
1.4.3 Build issues

### ISSUE THREAD

**Issue by podstanar0** [socram8888/tonyhax#138]
Hey, followed your development guide and when on final step i run `make`, this is the output error:
```
mips-linux-gnu-ld: secondary.o: in function `try_boot_cd':
secondary.c:(.text.try_boot_cd+0x224): undefined reference to `memset'
mips-linux-gnu-ld: secondary.o: in function `start':
secondary.c:(.start+0x20): undefined reference to `__memset_chk'
mips-linux-gnu-ld: bios.o: in function `bios_reinitialize':
bios.c:(.text.bios_reinitialize+0x20): undefined reference to `memset'
mips-linux-gnu-ld: debugscreen.o: in function `debug_write':
debugscreen.c:(.text.debug_write+0xa8): undefined reference to `__strcpy_chk'
make[1]: *** [Makefile:35: secondary.elf] Error 1
```
Btw i'm on Windows 10 using WSL

**Comment by socram8888** [comment]
What distro and version of Linux are you using? Looks like your GCC compiler isn't inlining a memset, which mine does.

**Comment by podstanar0** [comment]
Hey, i'm using default one (Ubuntu 20.04)

**Comment by socram8888** [comment]
Fixed in commit 8934a2bb614973064e45a6638e03fd0c7426e5fc

**Comment by podstanar0** [comment]
great, thanks
